packs/data/feats.db/electrify-armor.json

```diff
@@ -11,15 +11,15 @@
             "value": 1
         },
         "description": {
-            "value": "<p><strong>Requirements</strong> You are wearing your armor innovation.</p>\n<p>You electrify your armor to punish foes who dare to attack you. For 1 round, any creature that touches you, or that hits you with a melee unarmed attack or a non-reach melee weapon attack, takes [[/r {1d4}[electricity]]]{1d4 electricity damage}. The effect ends if you cease wearing your armor innovation.</p>\n<p>If you have the revolutionary innovation class feature, the damage increases to [[/r {2d4}[electricity]]]{2d4}.</p>\n<p><strong>Unstable Function</strong> You create an unstable chain reaction, sending countless sparks dancing across your armor. Add the unstable trait to Electrify Armor. The effects last for 1 minute instead of 1 round, and the damage dice increase from d4s to d12s.( [[/r {1d12}[electricity]]]{1d12}]] / [[/r {2d124}[electricity]]]{2d12})</p>"
+            "value": "<p><strong>Requirements</strong> You are wearing your armor innovation.</p>\n<p>You electrify your armor to punish foes who dare to attack you. For 1 round, any creature that touches you, or that hits you with a melee unarmed attack or a non-reach melee weapon attack, takes [[/r {1d4}[electricity]]]{1d4 electricity damage}. The effect ends if you cease wearing your armor innovation.</p>\n<p>If you have the revolutionary innovation class feature, the damage increases to [[/r {2d4}[electricity]]]{2d4}.</p>\n<p><strong>Unstable Function</strong> You create an unstable chain reaction, sending countless sparks dancing across your armor. Add the unstable trait to Electrify Armor. The effects last for 1 minute instead of 1 round, and the damage dice increase from d4s to d12s.( [[/r {1d12}[electricity]]]{1d12} / [[/r {2d12}[electricity]]]{2d12})</p>\n<p>@Check[type:flat|dc:17|name:Unstable Flat Check|showDC:all|traits:unstable]{Unstable Check}</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Requirements</strong> You are wearing your armor innovation.</p>\n<p>You electrify your armor to punish foes who dare to attack you. For 1 round, any creature that touches you, or that hits you with a melee unarmed attack or a non-reach melee weapon attack, takes [[/r {1d4}[electricity]]]{1d4 electricity damage}. The effect ends if you cease wearing your armor innovation.</p>\n<p>If you have the revolutionary innovation class feature, the damage increases to [[/r {2d4}[electricity]]]{2d4}.</p>\n<p><strong>Unstable Function</strong> You create an unstable chain reaction, sending countless sparks dancing across your armor. Add the unstable trait to Electrify Armor. The effects last for 1 minute instead of 1 round, and the damage dice increase from d4s to d12s.( [[/r {1d12}[electricity]]]{1d12} / [[/r {2d12}[electricity]]]{2d12})</p>\n<p>@Check[type:flat|dc:17|name:Unstable Flat Check]{Unstable Check}</p>"
```